Add EVM integration to docs (#202)
timewave-computer/valence-protocol: docs/src/authorizations_processors/processor_contract.md

# Processor Contract

The `Processor` will be a contract on each domain of our workflow. It handles the execution of the `Message Batches` that are sent to it by the `Authorization` contract. Depending on the

> type of `Processor` being used, the features will vary. We currently have two types of processors: `LiteProcessor` and `Processor`. The former is a simplified version of the latter, with more limited functionality and optimized for specific domains where gas costs are a concern.

Reviewer: ```suggestion
Depending on the `Processor` type in use, its features will vary. There are currently two types of processors: `Lite Processor` and `Processor`. The former is a simplified version of the latter. The `Lite Processor` has limited functionality to optimize for gas-constrained domains.
```
